internal/test/test.go

```diff
@@ -47,3 +49,16 @@ func CleanupConfig() {
 	config.SetOrgID("")
 	config.SetProjectID("")
 }
+
+// VerifyTemplate validates that the given template string is valid.
+func VerifyOutputTemplate(t *testing.T, templateString string, expected interface{}) {
+	t.Helper()
+	parsedTemplate, err := template.New("output").Parse(templateString)
+	if err != nil {
+		t.Fatalf("Failed to validate table format template: %v", err)
+	}
+	err = templatecheck.CheckText(parsedTemplate, expected)
+	if err != nil {
+		t.Fatalf("Failed to validate table format template: %v", err)
+	}
```

Comment:
FYI since you have fatal here the t.Log you usually run after won't be called

Reply:
Good point. Based on that - do we care to actually print rendered template to the end users. If no then I will remove logger for buffer. In most of the cases I did moved it to be the first print after this asert.

Reply:
I have no strong opinion on any way, either loosing this to `t.Errorf`, moving the logger to be before we call this function or deleting the logger, they all seems valid with the information I have

Reply:
Addressed outside this function by moving one print that happens after fatal. 
I think it is good to have fatal being part of the test (it matches definition of the test helpers)